test/apparmor.bats

```diff
@@ -153,3 +153,41 @@ function teardown() {
     cleanup_pods
     stop_ocid
 }
+
+# 5. test running with default apparmor profile unloaded.
+# test that we can will fail when running a ctr with rong apparmor profile name.
```

Comment:
My concern with this test is that if `ocid-default` is unloaded then we will reload our own profile over it. Which sounds like that's fine, but I'm not sure.

I would actually like it if we checked that the checksum (`/sys/kernel/security/apparmor/policy/profiles/ocid-default.1/sha1`). But that's something to consider later.